content/docs/reference/pkg/kubernetes/helm/v2/chart.md

```diff
@@ -2,16 +2,17 @@
 ---
 title: "Chart"
 title_tag: "Resource Chart | Module helm/v2 | Package Kubernetes"
-meta_desc: "Explore the Chart resource of the helm/v2 module, including examples, input properties, output properties, lookup functions, and supporting types. ConfigFile creates a set of Kubernetes resources from Kubernetes YAML file. If `config.name`"
+meta_desc: "Explore the Chart resource of the helm/v2 module, including examples, input properties, output properties, lookup functions, and supporting types. Chart is a component representing a collection of resources described by an arbitrary Helm Chart. The Chart can be fetched from any source that is accessible to the `helm` command line. Values in the `values.yml` file can be overridden using `ChartOpts.values` (equivalent to `--set` or having multiple `values.yml` files). Objects can be transformed arbitrarily by supplying callbacks to `ChartOpts.transformations`."
```

Comment:
Hmm I think we should update the description of the resource so that this sentence is on a different line and wouldn't show up here in the meta description.

```
The Chart can be fetched from any source that is accessible to the `helm` command line. Values in the `values.yml` file can be overridden using `ChartOpts.values` (equivalent to `--set` or having multiple `values.yml` files). Objects can be transformed arbitrarily by supplying callbacks to `ChartOpts.transformations`.
```

Reply:
I added suggestions below that control this property here.